api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/BaseDaoImpl.java

```diff
@@ -424,6 +431,62 @@ protected boolean containsAlias(Iterator<CriteriaImpl.Subcriteria> subcriteriaIt
 		return Optional.empty();
 	}
 	
+	protected Optional<Criterion> handleQuantity(String propertyName, QuantityParam quantityParam) {
+		if (quantityParam == null) {
+			return Optional.empty();
+		}
+		
+		BigDecimal value = quantityParam.getValue();
+		BigDecimal approxRange = APPROX_RANGE.multiply(value);
+		
+		if (quantityParam.getPrefix() == null) {
```

Comment:
Change this to `quantityParam.getPrefix() == null || quantityParam.getPrefix.equals(APPROXIMATE)`